src/code-snippet/code-snippet.component.ts

```diff
@@ -310,7 +310,7 @@ export class CodeSnippet extends BaseIconButton implements OnInit, AfterViewInit
 
 	protected canExpand() {
 		if (this.display === "multi") {
-			const height = this.codeContent.nativeElement.getBoundingClientRect().height;
+			const height = this.codeContent?.nativeElement.getBoundingClientRect().height;
```

Comment:
```suggestion
			const height = this.codeContent.nativeElement.getBoundingClientRect().height;
```

I think there is a bigger underlying issue than this, the view children aren't being picked up.

The biggest issue is that the `#code` directive is within a `ng-template` so it isn't eing picked up by Angular. Could you move that outside of the ng-template?

Before:
```html
		<ng-template #codeTemplate>
			<code #code><ng-content></ng-content></code>
		</ng-template>
```


After:
```
		<ng-template #codeTemplate>
			<ng-content></ng-content>
		</ng-template>
```

Then you will have to update all instances where that template is used and wrap it in `code` tag as such:
Inline with no copy:
```html
					<code #code>
						<ng-container *ngTemplateOutlet="codeTemplate"></ng-container>
					</code>
```

Multi:
```html
				<pre
					#codeContent
					*ngIf="!skeleton"
					(scroll)="(display === 'multi' ? handleScroll() : null)"><code #code><ng-container *ngTemplateOutlet="codeTemplate"></ng-container></code></pre>
```

Inline:
```html
					<code #code>
						<ng-container *ngTemplateOutlet="codeTemplate"></ng-container>
					</code>
```

Reply:
Sorry, I was on vacation - I don't see any of the files changed in the push 😅

Reply:
Could you push these changes and I can suggest additional changes/make updates

Reply:
Hey, hope you had a great vacation!
Sure, pushed my changes now.